src/sasatom.h

```diff
@@ -89,6 +91,77 @@ fetch_and_add(void *pointer, long int delta)
 }
 
 /*!
+ * Atomic fetch and and operation on memory \a pointer of int type.
+ *
+ * Performs the atomic operation:
+ * { tmp = *pointer; *pointer = tmp & delta; return *pointer }
+ *
+ * Returns \a *pointer before update.
+ */
+static inline long int
+fetch_and_and(unsigned int *pointer, int delta)
+{
+#if GCC_VERSION >= 40700
```

Comment:
Checked in other libraries.Availability of built-in functions are handled using gcc version.